src/pyrad_proc/pyrad/io/mxpol_config.py

```diff
@@ -0,0 +1,151 @@
+# -*- coding: utf-8 -*-
```

Comment:
I am not sure about the purpose of this file. I guess you can leave it as it is.